src/core/Form/types.ts

```diff
@@ -13,3 +13,24 @@ export interface StatusTextCommonProps {
    */
   statusTextAriaLiveMode?: AriaLiveMode;
 }
+
```

Comment:
Just a note that would it be useful to also export other types out from the library?

Reply:
Certainly worth considering 👍